### PR TITLE
Fix/fr welcome page

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Change welcome page for FR subsidiary
MBE-159

Needs #864 to be merged

### CODE REVIEW

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: client/app/configuration/configuration.html [hunk 1]
```diff
@@ -1,32 +1,52 @@
...
+                        class="oui-tile__item oui-tile__item_button"
+                        data-ng-repeat="link in ::section.links track by $index">
+                        <a
+                            target="_blank"
+                            class="oui-tile__button oui-button oui-button_link oui-button_icon-right oui-button_full-width"
+                            data-ng-href="{{:: link.url }}"
+                            data-at-internet-click="{ name:  link.atInternetClickTag }">
+                            <span data-translate="{{:: link.title }}"></span>
+                            <span
+                                class="fa fa-external-link my-auto ml-auto"
```
Comment (marie-j) [inline client/app/configuration/configuration.html]:
```suggestion
                                 class="oui-icon oui-icon-external_link my-auto ml-auto"
```

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_3#651.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_3#652.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Pull request merged**